### DISCUSSION TITLE
Documentation about supervisord is not accurate

### DISCUSSION THREAD

**Discussion by macropay-solutions** [laravel/framework#49633] [General]
As explained here https://stackoverflow.com/questions/76462520/how-to-handle-sigterm-signal-on-laravel-8-9-10-deploy-with-docker-containers-in the supervisord starts a new worker imediatelly after the SIGTERM is received by laravel and the worker finished its ongoing job and quits because supervisord is not aware of SIGTERM.

This might lead to jobs being not executed 100% in the time between the worker is being stopped and the container is being stopped (deploy or scale down scenario).
(an idempotent job would solve that but anyway, the documentation is not accurate)

I propose that the laravel documentation is updated in this regard. Supervisord is not a bulletproof solution.

**Comment by shaedrich** [comment]
You could create a PR in the respective repository: https://github.com/laravel/docs

**Reply by macropay-solutions** [reply]
A proper change would imply also an alternative not just a badge: "this is not good enough" and I'm not a devops, so not the right person to do it.

**Comment by ghost** [comment]
I found an alternative solution to supervisord.

https://github.com/laravel/framework/discussions/49954#discussioncomment-8351395

A new container with this script as an entrypoint in its task definition coupled with essential=false
